nusight2/src/client/components/vision/network.ts

```diff
@@ -128,8 +120,8 @@ export class VisionNetwork {
         goal.side === message.vision.Goal.Side.LEFT
           ? 'left'
           : goal.side === message.vision.Goal.Side.RIGHT
-          ? 'right'
-          : 'unknown',
+            ? 'right'
```

Comment:
Why did the formatting here change? Indented nested ternaries are from an older version of Prettier.